adafruit_wiznet5k/adafruit_wiznet5k_socket.py

```diff
@@ -335,10 +450,17 @@ def recv(self, bufsize=0, flags=0):  # pylint: disable=too-many-branches
         gc.collect()
         return ret
 
-    def embed_recv(self, bufsize=0, flags=0):  # pylint: disable=too-many-branches
-        """Reads some bytes from the connected remote address and then return recv().
-        :param int bufsize: Maximum number of bytes to receive.
+    def embed_recv(
+        self, bufsize: int = 0, flags: Any = 0
```

Comment:
another spot where I think we should match the `flags` type declaration with docstrings.